sources/naturalearth.json

```diff
@@ -3,9 +3,11 @@
 	"fullname": "Natural Earth",
 	"name": "naturalearth",
 	"prefix": "ne",
-	"key":"id",
+	"key": "id",
```

Comment:
Leave the `key` property - if it doesn't have a suitable key then leave the value empty. This was probably done at a time when you were adding/generating IDs for NE, if memory serves.